codegen/bin/coins

```diff
@@ -29,8 +25,7 @@ def self.explorer_account_url(c)
 end
 
 json_string = File.read('coins.json')
-coins = JSON.parse(json_string).sort_by { |x| x['name'] }
-coins_md = coins.sort_by { |x| Integer(coin_type(x['derivationPath'])) }
+coins = JSON.parse(json_string).sort_by { |x| x['numericId'] }
```

Comment:
Any particular reason to sort by numericId instead of name?  Human search is mostly by name.

Reply:
you meant this page (https://github.com/trustwallet/wallet-core/blob/master/docs/coins.md)? same as https://github.com/satoshilabs/slips/blob/master/slip-0044.md, people wont' other code generated code

Reply:
I meant the coins.json.  Diff is also more managable, if order is not changed there.

Reply:
it was not sorted at all, the first item is nebulas...